pages/projects/osg-software.md

```diff
@@ -24,25 +24,58 @@ software systems that comprise the OSG fabric of production services.
     abandoned software, and develops tools to fill any gaps to provide a software stack suitable for WLCG sites.
     This software stack is distributed through packages used widely by the community, such as RPMs and container images.
 
--   **Integration testing:** the team coordinates tests of the integrated software stack among internal staff, external
-    stakeholders, and automated means.
+-   **Integration testing:** the team coordinates tests of the integrated software stack among internal staff and
+    external stakeholders.
+    Additionally, the team maintains a nightly integration test suite that covers different combinations of software and
```

Comment:
Additionally, the team maintains and monitors nightly automated tests that exercise different integrated software installations, varying software versions, installation methods, and hosting operating systems.